src/Provider/TileProvider.js

```diff
@@ -9,20 +9,18 @@ import TileMesh from '../Core/TileMesh';
 import LayeredMaterial from '../Renderer/LayeredMaterial';
 import CancelledCommandException from '../Core/Scheduler/CancelledCommandException';
 import Cache from '../Core/Scheduler/Cache';
-import { requestNewTile } from '../Process/TiledNodeProcessing';
 
 function preprocessDataLayer(layer, view, scheduler) {
     if (!layer.schemeTile) {
         throw new Error(`Cannot init tiled layer without schemeTile for layer ${layer.id}`);
     }
 
     layer.level0Nodes = [];
-    layer.onTileCreated = layer.onTileCreated || (() => {});
 
     const promises = [];
 
     for (const root of layer.schemeTile) {
-        promises.push(requestNewTile(view, scheduler, layer, root, undefined, 0));
+        promises.push(layer.requestNewTile(view, scheduler, root, undefined, 0));
```

Comment:
Sorry, it's not a good idea for the moment because it should passe view and scheduler in constructor. You have to wait for the `tileMeshFromExtent (extent, builderGeometry)` function from `tileProvider`